CHANGELOG.rst

```diff
@@ -87,6 +87,7 @@ v0.7.0 RC2
    * in Chainsolve Transmutation, rerun example to update output
    * pytaps to pymoab update in intro tutorial (#1282)
    * in material_library example, updated import and class usage to reflect MaterialLibrary changes (#1280)
+   * adds broken yt warning to examples notebooks, updates deprecated urlretrieve to requests (#1295)
```

Comment:
should this be two separate change log entries?

Reply:
everyone is making the urlretrieve changes on their own w indiviual notebooks,so I don't know!

Reply:
ah- nbd it's probably fine to leave as is. These two statements separated by a comma are just seeming unrelated which is why I suggest two different entries.